src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -41,7 +41,7 @@ Astro's focus on content from its beginnings as a static site builder have allow
 
 ### Server-first
 
-**Astro leverages [server-rendering](/en/basics/rendering-modes/) over client-side rendering in the browser as much as possible.** This is the same approach that traditional server-side frameworks -- PHP, WordPress, Laravel, Ruby on Rails, etc. -- have been using for decades. But you don't need to learn a second server-side language to unlock it. With Astro, everything is still just HTML, CSS, and JavaScript (or TypeScript, if you prefer).
+**Astro leverages [server-rendering](/en/basics/rendering/) over client-side rendering in the browser as much as possible.** This is the same approach that traditional server-side frameworks -- PHP, WordPress, Laravel, Ruby on Rails, etc. -- have been using for decades. But you don't need to learn a second server-side language to unlock it. With Astro, everything is still just HTML, CSS, and JavaScript (or TypeScript, if you prefer).
```

Comment:
I did change `server-rendering` to not have a hyphen on the other page, but looks like we've been using it for a while, so pointing out I am not sure if that's grammatically correct.

Reply:
Thank you! I will do a global search and check for this throughout!